### DISCUSSION TITLE
Is there a way to access a database with nuetralinojs?

### DISCUSSION THREAD

**Discussion by JC3400** [neutralinojs/neutralinojs#733] [Q&A]
Is there a way to access a database (e.g. oracle) with nuetralinojs?

**Comment by shalithasuranga** [comment]
Hey.. yeah. you can write any custom backend code with any programming language with Neutralinojs's extensions API. Please check more details here:https://neutralino.js.org/docs/how-to/extensions-overview

**Comment by hschneider** [comment]
Meanwhile there exist some ready to plugin extensions:

https://github.com/neutralinojs-community/awesome-neutralino?tab=readme-ov-file#extensions
